app/src/main/java/com/example/capstone/JoinedSessionsAdapter.java

```diff
@@ -0,0 +1,119 @@
+    private List<StudySession> studySessions;
+
+    public JoinedSessionsAdapter(Context context, List<StudySession> studySessions) {
+        this.context = context;
+        this.studySessions = studySessions;
+    }
+
+    @NonNull
+    @Override
+    public ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
+        View view = LayoutInflater.from(context).inflate(R.layout.item_joined_study_session, parent, false);
+        return new ViewHolder(view);
+    }
+
+    @Override
+    public void onBindViewHolder(@NonNull ViewHolder holder, int position) {
+        StudySession studySession = studySessions.get(position);
+        try {
+            holder.bind(studySession);
+        } catch (ParseException e) {
+            e.printStackTrace();
+        }
+    }
+
+    @Override
+    public int getItemCount() {
+        return studySessions.size();
+    }
+
+    class ViewHolder extends RecyclerView.ViewHolder implements View.OnClickListener {
+        private FirebaseAuth mAuth;
+
+        private TextView tvStudySessionName;
+        private ImageView ivOrganizerProfilePicture;
+        private TextView tvStartTime;
+
+        public Uri getUserProfilePicture() throws ParseException {
+            FirebaseUser user = mAuth.getCurrentUser();
+            String firebase_uid = user.getUid();
+            ParseQuery<User> query = ParseQuery.getQuery(User.class);
+            query.include(User.KEY_FIREBASE_UID);
+            query.include(User.KEY_PROFILE_PICTURE);
+            query.whereEqualTo(User.KEY_FIREBASE_UID, firebase_uid);
+            Log.i(TAG, query.getFirst().getProfilePicture().getUrl());
+            return Uri.parse(query.getFirst().getProfilePicture().getUrl());
+        }
+
+        public ViewHolder(@NonNull View itemView) {
+            super(itemView);
+            itemView.setOnClickListener(this);
+            mAuth = FirebaseAuth.getInstance();
+            tvStudySessionName = itemView.findViewById(R.id.tv_study_session_name);
+            ivOrganizerProfilePicture = itemView.findViewById(R.id.iv_organizer_profile_picture);
+            tvStartTime = itemView.findViewById(R.id.tv_study_session_start_time);
+        }
+
+        public void bind(StudySession studySession) throws ParseException {
+            // Bind the post data to the view elements
+            tvStudySessionName.setText(studySession.getName());
+            tvStartTime.setText(studySession.getStartTime());
+
+            Uri profilePictureUrl = getUserProfilePicture();
+
+            if (profilePictureUrl != null) {
+                Glide.with(context).load(profilePictureUrl).into(ivOrganizerProfilePicture);
+            }
+        }
+
+        @Override
+        public void onClick(View v) {
+        }
+    }
+
+    public void clear() {
+        studySessions.clear();
+        notifyDataSetChanged();
+    }
+
+    // Add a list of items -- change to type used
+    public void addAll(List<StudySession> list) {
```

Comment:
These `addAll` and `clear` methods do not seem to be used anywhere. Please remove if unused